test/basicfuns.jl

```diff
@@ -97,8 +98,10 @@ end
     @test logaddexp(10002, 10003) ≈ 10000 + logaddexp(2.0, 3.0)
 
     @test logsumexp([1.0, 2.0, 3.0])          ≈ 3.40760596444438
+    @test logsumexp_onepass([1.0, 2.0, 3.0])  ≈ 3.40760596444438
```

Comment:
No need to test this now that it's the default. Same below: better test the public-facing function.